### PR TITLE
Bring README.md in line with actual files

### PR DESCRIPTION
**Pull request by matijs**
- update devDependencies
- `@babel/plugin-proposal-numeric-separator` was missing from  
the `.babelrc` example in `README.md`
- Update the `devDependencies` section in `README.md` to reflect  
 the changes after updating packages to the latest versions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,6 +75,7 @@ Then copy the `.babelrc` in this repo, or the below:
         "@babel/typescript"
     ],
     "plugins": [
+        "@babel/plugin-proposal-numeric-separator",
```
Comment (hg-pyun) [inline README.md]: I think `plugin-proposal-numeric-separator` should be optional.

Comment (matijs) [reply README.md]: Even better :) I have removed it completely.

Comment (hg-pyun) [reply README.md]: Looks good.

**Comment by orta** [comment]
Alright, there we go - this is now mergeable! Thanks all

**Pull request merged**